modules/extensions-mule-support/src/main/java/org/mule/runtime/module/extension/mule/internal/loader/parser/MuleSdkApplicationExtensionModelParser.java

```diff
@@ -47,6 +52,11 @@ public MuleSdkApplicationExtensionModelParser(String extensionName,
     this.extensionName = extensionName;
   }
 
+  @Override
+  public List<ModelProperty> getAdditionalModelProperties() {
+    return singletonList(new SdkFlavorModelProperty(SDK_FLAVOR_MULE_IN_APP));
```

Comment:
this should not come as an additional property. `ExtensionModelParser` should be given an explicit `ExtensionDevelopmentFramework getExtensionDevelopmentFramework()` method. Adding the model property should be done on the loader delegate.

Reply:
Done